server/routes/helpers.js

```diff
@@ -59,10 +59,76 @@ const prepareCourtRoomFilters = (allCourtRooms) => {
   return courtRooms
 }
 
+const getPaginationObject = (pageParams) => {
+  const maximumPages = 4
+
+  const { page: currentPage, matchingRecordsCount, recordsPerPage, courtCode, caseId, hearingId, defendantId } = pageParams
+
+  let startNum = Math.round(Math.ceil(currentPage - ((maximumPages - 1) / 2)))
```

Comment:
startNum is assigned here then potentially modified twice in the next 10 lines. Can we just assign it once? It might be easier to read the code that way
Same goes for endNum